src/python/dxpy/bindings/dxapplet.py

```diff
@@ -92,6 +92,7 @@ def _get_run_input_common_fields(executable_input, **kwargs):
 
         if kwargs.get('ignore_reuse') is not None:
             run_input["ignoreReuse"] = kwargs['ignore_reuse']
+        from ..utils.resolver import is_project_id
```

Comment:
https://github.com/dnanexus/dx-toolkit/blob/dae94b5e6f70340efeb22ad76f1f57ef9eab7b91/src/python/dxpy/bindings/dxapplet.py#L32
already imported here

Reply:
Yes, I removes this, It happened due to the rebase - I tried some code updates in this part but the code was already 'fixed' on the main branch.